app/views/search/_person.html.haml

```diff
@@ -3,11 +3,6 @@
     .media
       .media-left
         = profile_image_tag person
-        - if Rails.env.development? || current_user.super_admin?
-          %div.core-16
-            Score:
-            = hit._score
```

Comment:
do you know what this "hit" thing is? Just wondering if `_score` is used at all anywhere else and whether there are other things in the code that can be removed as a consequence of getting rid of this bit in the view. But I don't really know the app well enough to be sure.